src/components/views/rooms/MessageComposerInput.js

```diff
@@ -523,13 +523,27 @@ export default class MessageComposerInput extends React.Component {
         let sendTextFn = this.client.sendTextMessage;
 
         if (contentText.startsWith('/me')) {
-            contentText = contentText.substring(4);
+            if (contentText.startsWith('/me ')) {
+                contentText = contentText.substring(4);
+            } else if (contentText.startsWith('/me')) {
```

Comment:
@turt2live No it is a fix against typing just /me without it it would send a message which contains ```*<username> /me```